pkg/types/powervs/powervs_regions.go

```diff
@@ -65,3 +69,15 @@ var Regions = map[string]Region{
 		Zones:       []string{"us-east"},
 	},
 }
+
+// VPCRegionForRegion returns the VPC region for the specified region.
+func VPCRegionForRegion(region string) (string, error) {
+	var r Region
+	var ok bool
+
+	if r, ok = Regions[region]; ok {
+		return r.VPCRegion, nil
+	}
+
+	return "", fmt.Errorf("region %s not found in Regions", region)
```

Comment:
Error message can be improvised `fmt.Errorf("VPC region corresponding to a PowerVS region %s not found ", region)`